packages/create-astro/src/actions/template.ts

```diff
@@ -67,9 +67,14 @@ const FILES_TO_UPDATE = {
 };
 
 function getTemplateTarget(tmpl: string, ref = 'latest') {
+	if (tmpl === 'starlight') tmpl = 'starlight/basics';
+	const starlightMatches = tmpl.match(/^starlight\/(.+)$/);
+	if (starlightMatches) {
+		const [, starter] = starlightMatches;
+		return `withastro/starlight/examples/${starter}`;
+	}
```

Comment:
Just a suggestion for code clarity!
```suggestion
	if (tmpl.startsWith('starlight')) {
		const [_, starter = 'basics'] = tmpl.split('/');
		return `withastro/starlight/examples/${starter}`;
	}
```